tests/functional/spec/interviewer_note.spec.js

```diff
@@ -0,0 +1,32 @@
+import InitialInterstitialPage from "../generated_pages/interviewer_note/initial-interstitial-block.page.js";
```

Comment:
I believe we are keeping these in alphabetically order

Reply:
In order now, thanks!

Reply:
Done, thanks!